DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/Index.cshtml

```diff
@@ -207,12 +207,12 @@
               Other information:
             </dt>
             <dd class="nhsuk-summary-list__value">
-              @Model.OtherInformation
+              <partial name="_CentreStringArray" model="Model.OtherInformation" />
```

Comment:
I noticed that the GeneralInfo column in the Centres table had a few entries that split content onto multiple lines, so I've changed the display (and view model) to accommodate that.

Reply:
Nice!